programs/drift/src/controller/liquidation.rs

```diff
@@ -297,6 +297,21 @@ pub fn liquidate_perp(
         "liquidator_max_base_asset_amount cant be 0"
     )?;
 
+    validate_fill_price_within_price_bands(
```

Comment:
i think you can just call `is_oracle_too_divergent_with_twap_5min` since the oracle price is passed on the fill price and wont do anything